### PR TITLE
Padding things

### PR DESCRIPTION
**Pull request by lukewestby**
For #22 

I also added a couple missing type annotations to get my linter to stop complaining, and ran Css.elm throught elm-format.

Thanks!

### CODE REVIEW

File: src/Css.elm [hunk 1]
```diff
@@ -168,6 +168,7 @@ input =
     Tag "input"
 
 
+tagToString : Tag -> String
```
Comment (rtfeldman) [inline src/Css.elm]: Nice, thanks for adding annotations on these!

File: src/Css.elm [hunk 2]
```diff
@@ -1454,20 +1556,20 @@ introduceSelector selector declarations =
...
-                    . Foo
-                        ~ fontWeight bold
+                                stylesheet "homepage"
+                                    . Foo
+                                        ~ fontWeight bold
 
-            In this case, we don't make a new declaration, but rather add
-            the new selector to the existing list of selectors.
+                            In this case, we don't make a new declaration, but rather add
+                            the new selector to the existing list of selectors.
```
Comment (rtfeldman) [inline src/Css.elm]: Aha, looks like you encountered https://github.com/avh4/elm-format/issues/90 here :wink: - I've been deliberately excluding these from my commits, but maybe I should just switch this to use `--` comments until it's fixed...

**Comment by rtfeldman** [comment]
Looks great! Two things:

1. I went ahead and [swapped out that one problematic comment style](https://github.com/rtfeldman/elm-css/commit/3f57169784b6cc142e9fb3dcf56e8a5bd08160b8) so elm-format will stop generating spurious diffs. That'll cause a merge conflict, but you can just keep that version.
2. I've also merged a bunch of border stuff into `master`, so there will also be a conflict with the exports line if nothing else. Hopefully not too difficult to resolve!

File: src/Css.elm [hunk 3]
```diff
@@ -825,31 +833,125 @@ boxSizing =
...
 paddingBottom : Length -> ( String, String )
 paddingBottom =
     prop1 "padding-bottom" lengthToString
 
 
-{-| -}
+{-| Sets [`padding-right`](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-right)
+
+    paddingRight (px 10)
+-}
```
Comment (rtfeldman) [inline src/Css.elm]: :100: for adding docs to the existing ones!

File: src/Css.elm (hunk 2 above)
Comment (lukewestby) [inline src/Css.elm]: I thought it looked kind of odd! But I know there have been some changes and discussion around elm-format's style on elm-discuss recently so I wasn't sure. I'll keep an eye out for this in the future.

**Comment by lukewestby** [comment]
Merge conflicts resolved, thanks Richard!

**Comment by rtfeldman** [comment]
Awesome! :sunglasses:

**Pull request merged**